portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/pagestree/PagesTreeWithPermissions.testcase

```diff
@@ -172,4 +172,111 @@ definition {
 		}
 	}
 
+	@description = "This is a test for LPS-106110. Pages are not shown when users are not allowed to view certain pages."
+	@priority = "4"
+	test CannotViewPageWhenUserRemovingPermissions {
```

Comment:
@chaowang-123, let's call this `CannotViewChildPageWhenRemovingSiblingPagePermissions`